examples/atlas-encryptionAtRest-roles-two-step/second_step/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "public_key" {
+  description = "The public API key for MongoDB Atlas"
+  default     = ""
```

Comment:
The default values passed as an empty string just add more lines and are of no use. @themantissa if you think we can keep it, its ok. Else please remove these as well.